.changeset/gold-yaks-join.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/test-utils': patch
+---
+
+The test utility for PluginProvider called MockPluginProvider has been created. It will be handy in the cases when you use
+\_\_experimentalConfigure in your plugin
```

Comment:
```suggestion
The test utility for the plugin context called `MockPluginProvider` has been created. It will be handy in the cases when you use
`__experimentalConfigure` in your plugin. It is experimental and exported through `@backstage/test-utils/alpha`.
```